templates/cassiopeia/templateDetails.xml

```diff
@@ -105,30 +105,16 @@
 				</field>
 
 				<field
-					name="googleFont"
-					type="radio"
+					name="includeFont"
+					type="list"
 					label="TPL_CASSIOPEIA_FONT_LABEL"
-					description="TPL_CASSIOPEIA_FONT_DESC"
-					class="btn-group btn-group-yesno"
```

Comment:
better use layout="joomla.form.field.radio.switcher"

Reply:
Where are you looking? My PRv here changes that to a list field and removed that class.

Reply:
Can I close this conversation as resolved?

Reply:
yes